python/kserve/docs/V1beta1CatBoostSpec.md

```
@@ -0,0 +1,38 @@
+# V1beta1CatBoostSpec
+
+CatBoostSpec defines arguments for configuring CatBoost model serving.
+## Properties
+Name | Type | Description | Notes
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)